### PR TITLE
Replace `Study` with `sweep` in docs

### PR DESCRIPTION
**Pull request by 95-martin-orion**
From what I can tell, the nearest equivalent to the old concept of "Study" is a sweep. Strangely, the example code seems to be up-to-date for this.

### CODE REVIEW

File: docs/simulation.ipynb [hunk 1]
```diff
@@ -535,14 +535,12 @@
     "Here we see that the ``Symbol`` is used in two gates, and then the resolver provides this value at run time.\n",
     "\n",
     "Parameterized values are most useful in defining what we call a\n",
-    "``Study``.  A ``Study`` is a collection of trials, where each\n",
+    "``sweep``.  A ``sweep`` is a collection of trials, where each\n",
```
Comment (balopat) [inline docs/simulation.ipynb:538]: Sweep != Study

I think Study was a larger concept, see https://github.com/quantumlib/Cirq/commit/42faf4bc9e8d2d8acec696c1162923a5e9aecfa8#diff-d00dd130390b1095e7f1f95c4bd9ff4e20bb2d2b093674252d6b85dda0c3ba12R39

As such I think these sentences become confusing as Sweep is nothing else just an iterator over different values of the given parameters for a Circuit. Can you rephrase?

**Review by balopat — CHANGES_REQUESTED** [review]
Thanks for filing! I think we need to reflect the conceptual difference between study and sweep.

File: docs/simulation.ipynb (hunk 1 above)
Comment (balopat) [reply docs/simulation.ipynb:538]: Another thing: maybe we could capture the type of Sweeps somewhere - but that could be another PR. 
I described the different Sweep types here: https://github.com/quantumlib/Cirq/pull/3256#pullrequestreview-474013593 WDYT?

Comment (95-martin-orion) [reply docs/simulation.ipynb]: Cleaned this up a bit. Personally, I think "sweeps as a data structure" should be folded under the concept of subcircuits - a sweep is just a loop over an entire circuit with different parameters for each iteration, after all.

**Review by balopat — APPROVED** [review]
